libs/partners/ai21/langchain_ai21/ai21_base.py

```diff
@@ -12,7 +12,7 @@ class AI21Base(BaseModel):
     class Config:
         arbitrary_types_allowed = True
 
-    client: AI21Client = Field(default=None)
+    client: Any = Field(default=None, exclude=True)  #: :meta private:
```

Comment:
You can revert this, as this is fixed in the PR we have open in LC at the moment